### ISSUE TITLE
Unable to CALL stored procedure - MySQL [BUG] 

### ISSUE THREAD

**Issue by dleger** [cweijan/vscode-mysql#376]
- OS: Client on Fedora 34 and MySQL server on Ubuntu 20.04.2 LTS
- Database Type: MySQL 8.0.26

To reproduce this issue:
Create a simplistic table users with for instance only id and name

Then create this stored procedure:

```
DELIMITER //

CREATE PROCEDURE getusers()
BEGIN
SELECT * FROM users;
END //

DELIMITER ;
```

This works. I receive:
```
EXECUTE SUCCESS:

   CREATE PROCEDURE getusers() BEGIN SELECT * FROM users; END ;
```

Now open an "open query" window and do this:

CALL getusers();

I receive:
```

EXECUTE SUCCESS:

  CALL getusers()
```

But NO output: 暂无数据  (which means no data if I'm correct)

Clearly a bug, because when I do the same call directly from MySQL client in a terminal, I have the data.

Please could you look into it?

Cheers

**Comment by cweijan** [comment]
The current support for stored procedures is low, I will enhance it in the next version.

**Comment by cweijan** [comment]
Fix at version 4.3.0.

**Comment by dleger** [comment]
Brilliant!. I can now confirm this is  working.  :)
